src/Compilers/CSharp/Portable/BoundTree/BoundDecisionDag.cs

```diff
@@ -95,14 +95,12 @@ public BoundDecisionDag Rewrite(Func<BoundDecisionDagNode, Func<BoundDecisionDag
             // a node's successors before the node, the replacement should always be in the cache when we need it.
             var replacement = PooledDictionary<BoundDecisionDagNode, BoundDecisionDagNode>.GetInstance();
 
-            Func<BoundDecisionDagNode, BoundDecisionDagNode> getReplacementForChild = n => replacement[n];
```

Comment:
On the fence on removing this capture but it seems like it's purely to hide the underlying map. Doesn't seem to worth it.